lib/jellyfish/room.ex

```diff
@@ -437,6 +437,13 @@ defmodule Jellyfish.Room do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_info(%EndpointRemoved{endpoint_id: endpoint_id}, state) do
+    {_endpoint, state} = pop_in(state, [:peers, endpoint_id])
+    Logger.info("Peer #{endpoint_id} removed")
+    {:noreply, state}
+  end
```

Comment:
Also, we already have this case above? Or am I missing something?

Reply:
Yup, as I had a few breaks between writing both cases, I forgot about the previous one.